README.md

```diff
@@ -1 +1,33 @@
-# mongodb-operator
\ No newline at end of file
+# Charmed MongoDB Operator
+
+
+## Description
+
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database.
+
+MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB on physical or virtual machines as a single replica.
+
+
+## Usage
+
+Until the MongoDB Machine Charm is published, you need to follow the build & deploy instructions from [CONTRIBUTING.md](./CONTRIBUTING.md) to deploy the charm.
+
+
+## Relations
+
+Supported [relations](https://juju.is/docs/olm/relations):
+
+There are currently no supported relations.
+
+
+## License
+The Charmed MongoDB Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/MiaAltieri/mongodb-operator/master/blob/LICENSE) for more information.
```

Comment:
Here you can change the link to forward to./LICENSE directly to not forget to do it later when the repo will be transfered to Canonical organization.

Reply:
The problem is that the readme is also shown on Charmhub, and the link would then be broken on that site.

Reply:
Yeah, I realized this and I'm a bit unsure on what kind of links to use. Since the readme is shown on Charmhub, it would be better to use direct links to the repo. But the repo is not were it should be at the moment, so in any case it will need to be changed in the future.

Reply:
Understood. It definitely makes sense. Thank you Rob and Raul.